sql/rpl_rli.h

```diff
@@ -587,7 +588,7 @@ class Relay_log_info : public Slave_reporting_capability
     slave SQL thread is running, since the SQL thread reads it without
     a lock when executing Relay_log_info::flush().
   */
-  int sql_delay;
+  uint32_t &sql_delay= RelayLogInfoFile::sql_delay.value;
```

Comment:
This is unsigned in practice.
It was signed *only* because the parser uses `-1` to represent “unchanged”.
Only the lower half is configurable from CHANGE MASTER.